metagraph/src/cli/load/load_annotated_graph.cpp

```diff
@@ -62,6 +64,10 @@ std::unique_ptr<AnnotatedDBG> initialize_annotated_dbg(std::shared_ptr<DeBruijnG
             BinaryMatrix &matrix = const_cast<BinaryMatrix &>(annotation_temp->get_matrix());
             if (input_anno_type == Config::AnnotationType::RowDiff) {
                 dynamic_cast<RowDiff<ColumnMajor> &>(matrix).set_graph(dbg_graph);
+                std::string anchor_fname = config.infbase + ".anchors";
```

Comment:
this ".anchors" appears in lots of places. It makes sense to introduce a constant, like `RowDiff::kAnchExtension`

Reply:
A global constant works.
Alternatively, you can add a default BaseMatrix type and write `RowDiff<>::kAnchExtension`, which is not too long.